lib/pages/people/view/person_avatar.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:cached_network_image/cached_network_image.dart';
+import 'package:flutter/material.dart';
+
+class PersonAvatar extends StatelessWidget {
+  const PersonAvatar({@required this.photoURL, Key key}) : super(key: key);
+
+  final String photoURL;
+
+  @override
+  Widget build(BuildContext context) {
+    print('photoURL: $photoURL}');
+    return Padding(
+      padding: const EdgeInsets.all(12),
+      child: Container(
+        child: Center(
+          child: photoURL != null
+              ? CircleAvatar(
+                  radius: 100,
+                  backgroundImage: CachedNetworkImageProvider(photoURL),
+                )
+              : const CircleAvatar(
+                  child: Icon(
+                    Icons.person_outlined,
+                  ),
+                ),
+        ),
+      ),
+    );
+  }
```

Comment:
The case where the URL is null isn't gonna look right. Also, I think the Container and Center are unnecessary. I propose something similar to what we have on the home page profile card:

```suggestion
  Widget build(BuildContext context) {
    return Padding(
      padding: const EdgeInsets.all(12),
      child: CircleAvatar(
        radius: radius,
        backgroundImage:
            photoURL != null
                ? CachedNetworkImageProvider(photoURL)
                : const AssetImage(
                    'assets/illustrations/undraw_profile_pic.png',
                  ),
      )
    );
  }
```

Reply:
I removed them.